learning_resources/serializers_test.py

```diff
@@ -496,6 +496,8 @@ def test_content_file_serializer(settings, expected_types, has_channels):
                 "name": PlatformType[platform].value,
                 "code": platform,
             },
+            "published": content_file.published,
```

Comment:
Currently, we delete content files that are not published. Content files can be in the published=False state briefly so that we can delete them from the elastic search index before deleting the record. We don't need to add published to the serializer

https://github.com/mitodl/mit-learn/blob/index_run_content_files/learning_resources_search/indexing_api.py#L490